### PR TITLE
Readme features update

### PR DESCRIPTION
**Pull request by akiskips**
New Pull Request created for the Features paragraph in the main README, due to inconsistencies of the previous one [https://github.com/SAP/i40-aas/pull/97](url) after merging with canary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,21 +9,49 @@ __i40-aas__ welcomes [contributions](#contributing). Please read about [known is
...
+The AAS-Service provides an implementation of the RAMI 4.0 reference  architecture as specified by [Plattform Industrie 4.0](https://www.zvei.org/themen/industrie-40/details-of-the-asset-administration-shell/). Main features include:
+
+- Standardized and secure communication interfaces and API to access Asset Administration Shells of the digital representation of a physical asset (Digital Twin), that enable:
+  - use of a single AAS information model
+  - secure access to asset information from cloud and edge
+  - integration of assets without a communication interface (i.e. passive), e.g. via barcodes or QR-codes pointing to asset       management solutions
+
+- Facilitate Interoperability:
+  - The AAS-Service can act as a mediator between assets and applications that use different languages/standards
+  - Enable interactions between administration shells based on semantic protocols [(VDI/VDE 2193-1)](https://www.smart-production.de/open-automation/news-detailansicht/nsctrl/detail/News/vdivde-2193-einheitliche-sprache-fuer-industrie-40-komponenten-2019995/)
```
Comment (Manu3756) [inline README.md]: use the original source.
https://www.vdi.de/richtlinien/details/vdivde-2193-blatt-1-sprache-fuer-i40-komponenten-struktur-von-nachrichten

File: README.md [hunk 2]
```diff
@@ -9,21 +9,49 @@ __i40-aas__ welcomes [contributions](#contributing). Please read about [known is
...
+  - [Configuration](#Configuration)
+  - [Known Issues](#Known-Issues)
+  - [Get Support](#Get-Support)
+  - [Contributing](#Contributing)
+  - [Upcoming Changes](#Upcoming-Changes)
+  - [License](#License)
 
 
+## Main Features
+The AAS-Service provides an implementation of the RAMI 4.0 reference  architecture as specified by [Plattform Industrie 4.0](https://www.zvei.org/themen/industrie-40/details-of-the-asset-administration-shell/). Main features include:
```
Comment (Manu3756) [inline README.md]: new version https://www.plattform-i40.de/PI40/Redaktion/DE/Downloads/Publikation/Details-of-the-Asset-Administration-Shell-Part1.html

File: README.md [hunk 3]
```diff
@@ -9,21 +9,49 @@ __i40-aas__ welcomes [contributions](#contributing). Please read about [known is
...
+  - Enable interactions between administration shells based on semantic protocols [(VDI/VDE 2193-1)](https://www.smart-production.de/open-automation/news-detailansicht/nsctrl/detail/News/vdivde-2193-einheitliche-sprache-fuer-industrie-40-komponenten-2019995/)
+  - Information in the AAS can be exchanged between all
+  partners in a value chain (e.g. system manufacturers, operators, engineering and service partners, system integrators)
+
+- Provide registry and discovery services that:
+  -  hold digital models of various aspects submodels that describe the technical functionality exposed by the asset
+  -  implement endpoint handling for communication
+  -  unambiguously identify an asset in network
+
+- Easy implementation of complex business scenarios that support an asynchronous communication among AAS-enabled assets and applications (see [skills](https://github.com/SAP/i40-aas/blob/master/docs/markdown/onboarding-skill.md)).
```
Comment (Manu3756) [inline README.md]: We do not offer easy implementation of complex business scenarios. We offer skills that implement statefull interaction patterns for well defined or proprietary interactions.

File: README.md [hunk 4]
```diff
@@ -9,21 +9,49 @@ __i40-aas__ welcomes [contributions](#contributing). Please read about [known is
...
+
+- Enable integration of applications (eg. Asset stores, datalake) through [application adapters](https://github.com/SAP/i40-aas/blob/master/docs/markdown/adapter-registry.md)
+
+- Deploy everywhere (cloud, on-premise, edge) providing a simple developer experience with docker and K8s
+
+## Context - Industry 4.0
```
Comment (Manu3756) [inline README.md]: I do not think that we need this section. The image is enough to understand the context. We can move this to an own context.md file in docs folder if you think we should not just remove it. The image can stay.

File: README.md [hunk 5]
```diff
@@ -9,21 +9,49 @@ __i40-aas__ welcomes [contributions](#contributing). Please read about [known is
...
+  - [Upcoming Changes](#Upcoming-Changes)
+  - [License](#License)
 
 
+## Main Features
+The AAS-Service provides an implementation of the RAMI 4.0 reference  architecture as specified by [Plattform Industrie 4.0](https://www.zvei.org/themen/industrie-40/details-of-the-asset-administration-shell/). Main features include:
+
+- Standardized and secure communication interfaces and API to access Asset Administration Shells of the digital representation of a physical asset (Digital Twin), that enable:
+  - use of a single AAS information model
+  - secure access to asset information from cloud and edge
```
Comment (Manu3756) [inline README.md]: we do not have security full implemented yet. We should only list the current status here.

File: README.md (hunk 4 above)
Comment (akiskips) [reply README.md]: Ok, moved the section into a separate file in /docs.

File: README.md (hunk 3 above)
Comment (akiskips) [reply README.md]: Updated, im still not sure if the wording is the best with the new version. Please review the skills feature again

File: README.md (hunk 1 above)
Comment (akiskips) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (akiskips) [reply README.md]: updated

File: docs/markdown/Context-Industry40.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+## Context - Industry 4.0
+
+The implementation of the AAS-Service is related to the [Plattform Industrie 4.0](https://www.plattform-i40.de/PI40/Navigation/EN/Home/home.html) activities being driven by working groups in Germany. Especially the working group [Reference Architectures, Standards, and Norms](https://www.plattform-i40.de/PI40/Navigation/EN/ThePlatform/PlatformWorkingGroups/Reference-Architectures-Standards-Norms/reference-architectures-standards-norms.html) has defined a reference architecture and a [specification](https://www.plattform-i40.de/PI40/Redaktion/EN/Downloads/Publikation/2018-details-of-the-asset-administration-shell.pdf?__blob=publicationFile&v=5) to address a challenging topic to hardware and software vendors: __interoperability__. Especially the integration of assets (devices, machines, software, documents, etc.) into business processes is always cumbersome, time-consuming, and usually costly. As shown in the following figure, this service is meant to simplify the integration processes, and has further potential to extend existing applications.
```
[the remark this answers is not in the document]
Comment (akiskips) [reply docs/markdown/Context-Industry40.md]: Ok removed the reference.

**Pull request merged**